tox.ini

```diff
@@ -69,3 +64,12 @@ commands = bash {toxinidir}/tools/test/functest.sh
 
 [testenv:docs]
 commands = sphinx-build -j auto -d {toxinidir}/doc/build/doctrees -b html {toxinidir}/doc/source doc/build/html
+
+[testenv:coverage]
+commands =
```

Comment:
We need a `depends = py3` here, otherwise `tox` wouldn't be able to see that `coverage` env needs `py3` is needed to be run first.